Documentation/spelling_wordlist.txt

```diff
@@ -103,6 +103,8 @@ Polytechnique
 Portworx
 QCon
 QLogic
+Reconciler
+Reconcilers
```

Comment:
These aren't necessary given that you also added the lower-case (and thus, case-insensitive) version for these words.

Reply:
dropped.